src/libraries/System.Globalization/tests/Invariant/runtimeconfig.template.json

```diff
@@ -1,5 +1,6 @@
 {
   "configProperties": {
-    "System.Globalization.Invariant": true
+    "System.Globalization.Invariant": true,
+    "System.Globalization.PredefinedCulturesOnly": false
```

Comment:
Should we rather test `PredefinedCulturesOnly: true` since it is the new default?

Reply:
We already testing it in the new test case added in the file GetCultureInfo.cs. I want to keep this Invariant test be exercised with the old behavior to ensure we are not regressing either way.